frontend/src/basic/NavBar/NavBar.tsx

```diff
@@ -95,102 +102,64 @@ const NavBar = (): JSX.Element => {
     setOpen(closeAll);
   }, [page, setOpen]);
 
-  const slot = garage.getSlot();
-
   return (
     <Paper
-      elevation={6}
       sx={{
-        height: `${navbarHeight}em`,
-        width: `100%`,
         position: 'fixed',
         bottom: 0,
-        borderRadius: 0,
+        left: 0,
+        right: 0,
+        zIndex: 1000,
+        ...(isMobile
+          ? {
+              boxShadow: 'none',
```

Comment:
I see you already user `styled` up there, I believe is a good idea to create then at the bottom of every file where you add these objects to try to separate CSS from code

Reply:
Ok, I'll move them to the bottom, and will make a few more changes..

Reply:
@KoalaSat I have made the changes, please review it once more. Some of the things are hard coded for now as I was experimenting with the layout trying to figure out what works best, As I change other pages, I'll fix these too. But I guess it works for now.